dotCMS/src/main/java/com/dotcms/filters/NormalizationFilter.java

```diff
@@ -36,7 +36,12 @@ public String getRequestURI() {
                  2. A ".." segment is removed only if it is preceded by a non-".." segment.
                  3. Normalization has no effect upon opaque URIs. (mailto:a@b.com)
                  */
-                return URI.create(super.getRequestURI()).normalize().toString();
+                String newNormal = URI.create(super.getRequestURI()).normalize().toString();
```

Comment:
I think you could use something like
```
return newNormal.replaceAll(Pattern.quote("//"), StringPool.SLASH);
```

Reply:
While the code looks "cleaner", my thinking was that a replaceAll with a RegEx was actually going to be slower than just doing a string replace in a loop.